docs/help/security.mdx

```diff
@@ -0,0 +1,242 @@
+  [account settings](https://cloud.hasura.io/account-settings/access-tokens) on the Hasura Cloud dashboard or via the
+  DDN CLI with the `ddn auth login` command and `ddn auth print-pat` command.
+
+### Service Account Tokens
+
+Service Account Tokens are designed for applications, services, or automated processes that need to interact with the
+Hasura DDN Control Plane.
+
+- **Usage:** Ideal for continuous integration and continuous deployment (CI/CD) pipelines, automated scripts, and
+  backend services.
+- **Security:** Associated with a project, not an individual user.
+- **Generation:** You can create new service accounts and manage tokens for existing ones via your project's
+  `Settings` > `Service Accounts` tab of the console.
+
+## Securely Managing Secrets with Environment Variables
+
+Hasura DDN strongly recommends using environment variables to manage sensitive information, such as API keys, database
+connection strings, and other credentials. This practice enhances security by keeping secrets out of your codebase and
+metadata files.
+
+### Best Practices
+
+1.  **Use Environment Variables:** Store sensitive information in environment variables rather than hardcoding them
+    directly into your configuration files.
+
+2.  **Local Development:** For local development, use `.env` files to manage environment variables. The DDN CLI
+    automatically recognizes and loads these files.
+
+3.  **Cloud Deployment:** When deploying to Hasura DDN, use the project's `.env.cloud` file.
+
+4.  **Connector Configuration:** When configuring data connectors, use the `valueFromEnv` option to reference
+    environment variables. This ensures that sensitive values are loaded from the environment rather than being stored
+    directly in the configuration.
+
+```yaml
+# Example of using valueFromEnv in a connector configuration
+kind: DataConnectorLink
+version: v1
+definition:
+  name: my_connector
+  url:
+    readWriteUrls:
+      read:
+        valueFromEnv: MY_CONNECTOR_READ_URL # URL is stored as an env var
+      write:
+        valueFromEnv: MY_CONNECTOR_WRITE_URL # URL is stored as an env var
+  headers:
+    Authorization:
+      valueFromEnv: MY_CONNECTOR_AUTHORIZATION_HEADER # API Key/Secret is stored as an env var
+  schema: ...
+```
+
+5.  **Gitignore:** Ensure that your `.env` files, and any other files that might hold secrets, are added to your
+    `.gitignore` file. This will prevent these files from being committed to version control.
+
+### Metadata Immutability and Secret Protection
+
+Once a supergraph build is created, the metadata associated with that build, including any resolved values from
+environment variables, is immutable. Critically, **secrets incorporated into the build via environment variables are not
+directly exposed or retrievable from the built metadata once on the control plane**.
+
+You can retrieve a build using the `ddn supergraph build get <build-version>` command, however, this will not expose
```

Comment:
This statement is not very assuring (just saying that it will not expose secrets is not enough). Need to add more details on what this command does or doesnot.